akka-persistence-typed/src/main/scala/akka/persistence/typed/internal/Running.scala

```diff
@@ -343,6 +344,7 @@ private[akka] object Running {
               event.sequenceNumber,
               originReplicaId,
               expectedSequenceNumber)
+          event.replyTo.foreach(_ ! EventConsumed(event.persistenceId, event.sequenceNumber))
```

Comment:
wonder if we should also ack for the `originReplicaId == replication.replicaId` case, but maybe that isn't supposed to happen?